app/Http/Livewire/LdapSettingsForm.php

```diff
@@ -0,0 +1,244 @@
+      ];
+      protected  array $messages = [
+              'ldap_username_field.not_in' => '<code>sAMAccountName</code> (mixed case) will likely not work. You should use <code>samaccountname</code> (lowercase) instead. ',
+              'ldap_auth_filter_query.not_in' => '<code>uid=samaccountname</code> is probably not a valid auth filter. You probably want <code>uid=</code> ',
+              'ldap_filter.regex' => 'This value should probably not be wrapped in parentheses.',
+          ];
+
+
+
+    public Setting $setting;
+
+    public function mount(){
+
+        $this->setting = Setting::getSettings();
+        $this->groups = Group::all();
+
+        $this->ldap_enabled              = $this->setting->ldap_enabled;
+        $this->ldap_server               = $this->setting->ldap_server;
+        $this->ldap_server_cert_ignore   = $this->setting->ldap_server_cert_ignore;
+        $this->ldap_uname                = $this->setting->ldap_uname;
+        $this->ldap_pword                = $this->setting->ldap_pword;
+        $this->ldap_basedn               = $this->setting->ldap_basedn;
+        $this->ldap_default_group        = $this->setting->ldap_default_group;
+        $this->ldap_filter               = $this->setting->ldap_filter;
+        $this->ldap_username_field       = $this->setting->ldap_username_field;
+        $this->ldap_lname_field          = $this->setting->ldap_lname_field;
+        $this->ldap_fname_field          = $this->setting->ldap_fname_field;
+        $this->ldap_auth_filter_query    = $this->setting->ldap_auth_filter_query;
+        $this->ldap_version              = $this->setting->ldap_version;
+        $this->ldap_active_flag          = $this->setting->ldap_active_flag;
+        $this->ldap_emp_num              = $this->setting->ldap_emp_num;
+        $this->ldap_email                = $this->setting->ldap_email;
+        $this->ldap_manager              = $this->setting->ldap_manager;
+        $this->ad_domain                 = $this->setting->ad_domain;
+        $this->is_ad                     = $this->setting->is_ad;
+        $this->ad_append_domain          = $this->setting->ad_append_domain;
+        $this->ldap_tls                  = $this->setting->ldap_tls;
+        $this->ldap_pw_sync              = $this->setting->ldap_pw_sync;
+        $this->ldap_phone_field          = $this->setting->ldap_phone_field;
+        $this->ldap_jobtitle             = $this->setting->ldap_jobtitle;
+        $this->ldap_country              = $this->setting->ldap_country;
+        $this->ldap_dept                 = $this->setting->ldap_dept;
+        $this->ldap_client_tls_cert      = $this->setting->ldap_client_tls_cert;
+        $this->ldap_client_tls_key       = $this->setting->ldap_client_tls_key;
+        $this->custom_forgot_pass_url    = $this->setting->custom_forgot_pass_url;
+
+    }
+
+    public function render(){
+
+        return view('livewire.ldap-settings-form');
+    }
+
+
+    public function submit(){
```

Comment:
Even though we are checking on the front end, can we do the `if (! config('app.lock_passwords') === true) {` check here as well like we do in `SettingsController::postLdapSettings()`?